packages/monitor-v2/src/llm-bot/examples.ts

```diff
@@ -0,0 +1,114 @@
+import { Provider } from "@ethersproject/abstract-provider";
```

Comment:
I love this example -- I like your idea of creating more and more fully-formed examples as we build out these interfaces. +1